tools/start_server.py

```diff
@@ -146,9 +147,22 @@ def handle_save_ptu_to_file(self, _method_name, _params):
     file_name = _params["fileName"]
     content = json.loads(_params["content"])
 
-    with open(file_name, 'w') as json_file:
-      json.dump(content, json_file)
+    # Validate file path
+    def isFileNameValid(_file_name):
```

Comment:
Why use a sub function for something called once?

Reply:
Just to keep it organized. Similar to how we use lambdas in sdl_core https://github.com/smartdevicelink/sdl_core/blob/master/src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/mobile/get_interior_vehicle_data_consent_request.cc#L298

The file name validation isn't strictly related to saving the PTU file so it can be in its own function. I could create a private helper function in the class but since its only used by one function at the moment I created a sub-function for it